app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Api
+  module Verify
+    class DocumentCaptureController < BaseController
+      self.required_step = 'document_capture'
+      include ApplicationHelper
+
+      def create
+        result = @form ||= Idv::ApiDocumentVerificationForm.new(
+          verify_params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+          analytics: analytics,
+        ).submit
+
+        if result.success?
+          enqueue_job
+
+          render json: {
+            status: 'in_progress',
+          }, status: :ok
+        else
+          render json: { error: result.errors }, status: :bad_request
+        end
+      end
+
+      private
+
+      def enqueue_job
+        verify_document_capture_session = DocumentCaptureSession.
+          find_by(uuid: params[:document_capture_session_uuid])
+        verify_document_capture_session.requested_at = Time.zone.now
+        verify_document_capture_session.create_doc_auth_session
+
+        document_attributes = verify_params.to_h
+        applicant_pii = {}
+        applicant = {
+          user_uuid: applicant_pii[:uuid],
+          uuid_prefix: applicant_pii[:uuid_prefix],
```

Comment:
is this an outline for future work? If `applicant_pii` is always the empty hash, `applicant_pii[:uuid]` will always be nil

Reply:
it looked like `Idv::Agent.new(applicant).proof_document(` needed this hash in order to be used.

Reply:
can we add real values then instead of nil ones?

Reply:
Then again, based on some prior work (#5323), maybe these _should_ be real values, and if they aren't currently being populated, that's actually a bug?

Reply:
I spent more time digging into this: Turns out this is a real, existing bug with async document capture (not live in production). I filed a ticket at LG-6585, but we should be providing values for `user_uuid` and `uuid_prefix` as part of this implementation, corresponding to the user's `uuid` and the service provider's `app_id` (if applicable).